pkg/imgpkg/cmd/copy.go

```diff
@@ -7,10 +7,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/google/go-containerregistry/pkg/v1/types"
-	"github.com/k14s/imgpkg/pkg/imgpkg/imagelayers"
```

Comment:
sounds like we can actually delete imagelayers package as well?

Reply:
scratch that i remembered that it was being used by TarWriter. i think given its single use location, we should move contents of imagelayers into tar writer.

Reply:
I'll move the files to the tarwriter package. I do think there is value in keeping the logic of whether a layer should be written in its own struct and not be inlined into the tarwriter itself.